mne/preprocessing/tests/test_ica.py

```diff
@@ -1426,7 +1426,7 @@ def test_ica_labels():
     scores = ica.find_bads_muscle(raw)[1]
     assert 'muscle' in ica.labels_
     assert ica.labels_['muscle'] == [0]
-    assert_allclose(scores, [0.56, 0.01, 0.03, 0.00], atol=0.03)
+    assert_allclose(scores, [0.56, 0.01, 0.03, 0.00], atol=0.03)  # FIXME
```

Comment:
@alexrockhill this test is failing now, after a change to the default `n_fft` used to compute the welch spectrum (which gets used to compute the muscle PSD slope).  Where did these numbers / tolerance come from?  What is the right fix: making it more tolerant or adjusting the target to match what is actually computed?

cc @agramfort who approved #10534 where this was added.

Reply:
(NB: the test was failing *before* the change to use `expit` in `find_bads_muscle`.  That change is just for readability and shouldn't affect anything)

Reply:
these numbers are pretty arbitrary to me. What matters most if that we have `assert ica.labels_['muscle'] == [0]` so the first entry is the highest. Now if they change a lot it means something changed in the way signal is filtered. If it's just numerical noise then ICA can be very sensitive to the input so I could understand that ICA solution as slightly changed

Reply:
Sorry for the late reply, was at a conference. Yes, fine to relax the tolerance, the results just need to be the same,

Reply:
@agramfort I don't think I can say confidently that this is "just numerical noise". `scores are`:

```
array([0.56704471, 0.01110581, 0.02878223, 0.00165632])  # before n_fft change
array([0.46179387, 0.00936564, 0.02071585, 0.00222098])  # now
```

@alexrockhill 
> fine to relax the tolerance, the results just need to be the same

Do you consider the before/after to "be the same" here?  To me it seems that the numbers change a lot...  I have to change the `atol` from 0.03 to 0.1 to get it to pass again.  Given that some of the numbers are around 0.001, an atol of 0.1 seems huge.

Reply:
I meant qualitatively the same so I think that's fine. I might do `array([0.5, 0.01, 0.03, 0])` with an `atol=0.05`. I know that would cause the original test to fail but the user can set the threshold and, in my experience, there is a strongly bimodal distribution with muscle components >> 0.1 and non-muscle components << 0.1. The exact values aren't validated on a dataset yet although that would be a good thing to do at some point.